app/lib/file_parse_service.rb

```diff
@@ -109,6 +109,130 @@ def self.run_parse_job(study_file, study, user, reparse: false, persist_on_fail:
     end
   end
 
+  # handle setting up and launching a differential expression job
+  #
+  # * *params*
+  #   - +cluster_file+      (StudyFile) => Clustering file being used as control cell list
```

Comment:
Aligning type with other rows:

```suggestion
  #   - +cluster_file+     (StudyFile) => Clustering file being used as control cell list
```